samples/snippets/data_client/data_client_snippets_async.py

```diff
@@ -242,3 +242,24 @@ async def read_with_filter(project_id, instance_id, table_id):
                     print(row)
     # [END bigtable_async_reads_filter]
     await read_with_filter(table.client.project, table.instance_id, table.table_id)
+
+
+async def execute_query(table):
+    # [START bigtable_async_execute_query]
+    from google.cloud.bigtable.data import BigtableDataClientAsync
+
+    async def execute_query(project_id, instance_id, table_id):
+        async with BigtableDataClientAsync(project=project_id) as client:
+            query = (
+                "SELECT _key, os_build, connected_cell, connected_wifi "
+                f"from {table_id} WHERE _key=@row_key"
+            )
```

Comment:
I know I said to ignore the samples tests, but it looks like there's actually an issue here:
```
google.api_core.exceptions.InvalidArgument: 400 Syntax error: Missing whitespace between literal and alias [at 1:87]
```

Reply:
Also, there's a snippet lint issue that can be addressed using

```
cd samples/snippets/data_client
nox -s blacken
```

Reply:
To fix the invalid argument do: ```f"from `{table_id}` WHERE _key=@row_key"``` (wraps the table name in backticks)

Reply:
Daniel pointed out the column names here aren't working either. All of those are qualifiers.

We should update the first line of the query to `SELECT _key, stats_summary['os_build'], stats_summary['connected_cell'], stats_summary['connected_wifi'] `

Reply:
fixed.